### PR TITLE
Feat scaffold player

### PR DESCRIPTION
**Pull request by mixcas**
Player scaffolding
Adds basic play by click functionality

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -89,6 +92,55 @@ Site.Paypal = {
...
+
+  bind: function() {
+    var _this = this;
+
+    // Init youtube whuen youtube api is ready
+    // TODO: subscribe to this event with jQuery
+    window.onYouTubePlayerAPIReady = _this.initYoutube.bind(_this);
+
+    // Listen for updatedyoutubelist
+    $(window).on('updatedyoutubelist', function(event, data) {
```
Comment (mixcas) [inline src/js/main.js:122]: This stuff about `list` is really not used on this feat yet but i intend to use it to track the list of videos.

File: src/js/main.js [hunk 2]
```diff
@@ -89,6 +92,55 @@ Site.Paypal = {
...
+    // Bind stuff
+    _this.bind();
+
+  },
+
+  bind: function() {
+    var _this = this;
+
+    // Init youtube whuen youtube api is ready
+    // TODO: subscribe to this event with jQuery
```
Comment (varffs) [inline src/js/main.js:118]: I don't think this has to use jQuery if that is pretty standard code. But I'm pretty sure the jq bind is just `$(window).on('onYouTubePlayerAPIReady')` ?

Comment (mixcas) [reply src/js/main.js:118]: I think i tried

**Comment by varffs** [comment]
@mixcas I added youtube player vars to the Player object. Ready to merge now

**Pull request merged**